src/sentry/static/sentry/app/components/events/contextSummary.jsx

```diff
@@ -133,59 +159,32 @@ class EventContextSummary extends React.Component {
   render() {
     let evt = this.props.event;
     let contexts = evt.contexts;
-
-    let children = [<UserSummary key="user" data={evt.user} />];
-    switch (evt.platform) {
-      case 'cocoa':
-      // fallthrough
-      case 'native':
-        children.push(<DeviceSummary key="device" data={contexts.device} />);
-        children.push(
-          <GenericSummary key="os" data={contexts.os} unknownTitle={t('Unknown OS')} />
-        );
-        break;
-      case 'java':
-        if (contexts.os && contexts.os.name === 'Android') {
-          children.push(<DeviceSummary key="device" data={contexts.device} />);
-          children.push(
-            <GenericSummary key="os" data={contexts.os} unknownTitle={t('Unknown OS')} />
-          );
-        }
-        break;
-      case 'javascript':
-        children.push(
-          <GenericSummary
-            key="browser"
-            data={contexts.browser}
-            unknownTitle={t('Unknown Browser')}
-          />
-        );
-        children.push(
-          contexts.os ? (
-            <GenericSummary key="os" data={contexts.os} unknownTitle={t('Unknown OS')} />
-          ) : (
-            <DeviceSummary key="device" data={contexts.device} />
-          )
-        );
-        break;
-      default:
-        children.push(
-          <GenericSummary
-            key="runtime"
-            data={contexts.runtime}
-            unknownTitle={t('Unknown Runtime')}
-          />
-        );
-        children.push(
-          contexts.os ? (
-            <GenericSummary key="os" data={contexts.os} unknownTitle={t('Unknown OS')} />
-          ) : (
-            <DeviceSummary key="device" data={contexts.device} />
-          )
-        );
-        break;
+    let count = 0;
+
+    // Add defined contexts in the declared order, until we reach the limit
+    // defined by CONTEXT_COUNT_MAX.
+    let children = KNOWN_CONTEXTS.map(({key, Component, ...props}) => {
```

Comment:
I know this is from previous code but can we change the name of `children` - I think we should try to reserve it to refer to `children` from props.